private/detectchannelnan.m

```diff
@@ -0,0 +1,35 @@
+function [badchannels] = detectchannelnan(cfg,data)
+
+% DETECTCHANNELNAN detects whether one or more channels contain only NaNs
+% and if so adds them to cfg.badchannels
+%
+% Copyright (C) 2003, Robert Oostenveld
```

Comment:
I cannot claim copyright (and certainly not in 2003) for a function that you wrote.

Reply:
```suggestion
% Copyright (C) 2019, Johannes Algermissen
```